apps/judicial-system/backend/src/app/modules/notification/notification.service.ts

```diff
@@ -52,6 +52,7 @@ import { CaseEvent, EventService } from '../event'
 import { SendNotificationDto } from './dto/sendNotification.dto'
 import { Notification } from './models/notification.model'
 import { SendNotificationResponse } from './models/sendNotification.resopnse'
+import { formatReadyForCourtEmailNotificationToProsecutor } from '../../formatters/formatters'
```

Comment:
Consider adding the new export to the formatters barrel file and import it from there.

Reply:
Adressed 0f841917846b9d78c85b4332b274739406656f7a